src/orders/views/OrderTransactionRefundCreate/OrderTransactionRefundCreate.tsx

```diff
@@ -46,7 +47,26 @@ const OrderTransactionRefund: React.FC<OrderTransactionRefundCreateProps> = ({
           ),
         );
       }
+      if (submitData?.orderGrantRefundCreate?.errors.length) {
+        const errors = submitData.orderGrantRefundCreate.errors;
+        setLinesErrors(
+          errors.map(err => ({
+            code: err.code,
+            field: err.field,
+            lines: err.lines,
+            message: err.message,
+          })) as OrderTransactionRefundError[],
+        );
+        errors.forEach(err => {
+          if (err.code !== OrderGrantRefundCreateErrorCode.REQUIRED)
+            notify({
+              status: "error",
+              text: err.message,
+            });
+        });
+      }
     },
```

Comment:
```suggestion
      if (submitData?.orderGrantRefundCreate?.errors.length > 0) {
        const { errors } = submitData.orderGrantRefundCreate;
        const errorLines: OrderTransactionRefundError[] = [];

        errors.forEach(err => {
          if (err.code !== OrderGrantRefundCreateErrorCode.REQUIRED) {
            notify({
              status: "error",
              text: err.message,
            });
          }
          
          errorLines.push({
            code: err.code,
            field: err.field,
            lines: err.lines,
            message: err.message,
            } as OrderTransactionRefundError)
        });
                    
         setLinesError(errorLines);
    },
```

Suggestion: Totally optional but I think here we can just loop once 😄